recipes/qt/5.x.x/conanfile.py

```diff
@@ -265,13 +261,27 @@ def _enablemodule(mod):
                 _enablemodule(req)
 
         for module in self._submodules:
+            for status in self._module_statuses:
+                if getattr(self.options, f"{status}_modules"):
+                    if module in submodules_tree and submodules_tree[module]['status'] == status:
+                        _enablemodule(module)
             if self.options.get_safe(module):
```

Comment:
```suggestion
            module_enabled = False
            for status in self._module_statuses:
                if getattr(self.options, f"{status}_modules"):
                    if module in submodules_tree and submodules_tree[module]['status'] == status:
                        _enablemodule(module)
                        module_enabled = True
                        break
            if self.options.get_safe(module) and not module_enabled:
```
Avoid enabling twice